### ISSUE TITLE
why not support struct with generic?? quick_xml can do it!!!

### ISSUE THREAD

**Issue by xfront** [media-io/yaserde#187]
```
#[derive(Serialize, YaSerialize, Debug)]
    struct Envelop<T> {
        body: T,
    }

 struct Envelop<T> {
    |            ^^^^^^^ -
help: add missing generic argument
    |
147 |     struct Envelop<T><T> {
    |                   +++
```

**Comment by mtorromeo** [comment]
See #156

**Comment by antis81** [comment]
I ended up using quick_xml/serde for deserialization and yaserde for serialization (into a custom format). Serialization into formats other than XML is where yaserde shines. :sun_with_face:

**Comment by MarcAntoine-Arnaud** [comment]
Generics are now supported via https://github.com/media-io/yaserde/pull/189
It's available in latest published version (0.10.1)

**Comment by MarcAntoine-Arnaud** [comment]
I close the issue as I consider it's addressed. Re-open it if needed.
